questions.ts

```diff
@@ -484,17 +497,19 @@ export const askQuestions = async (
       // continue asking user for data sources if none are initially provided
       while (true) {
         const firstQuestion = program.dataSources.length === 0;
+        const choices = getDataSourceChoices(
```

Comment:
for LlamaParse, we're already asking for the llamacloud key:
```
 // Ask for LlamaCloud API key
      if (useLlamaParse && program.llamaCloudKey === undefined) {
        const { llamaCloudKey } = await prompts(
          {
            type: "text",
            name: "llamaCloudKey",
            message:
              "Please provide your LlamaIndex Cloud API key (leave blank to skip):",
          },
          questionHandlers,
        );
        program.llamaCloudKey = llamaCloudKey;
      }
      ```

also ask for this key for LlamaCloud datasource (it's the same key)